bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -58,13 +60,9 @@ trait ItcTable extends Table {
     val seq = parameters.sequence
     val allKeys = seq.getStaticKeys.toSeq ++ seq.getIteratedKeys.toSeq
     val showKeys = seq.getIteratedKeys.toSeq.
-      filterNot(_.equals(DATALABEL_KEY)). // don't show the original data label (step number)
-      filterNot(_.equals(OBS_TYPE_KEY)). // don't show the type (science vs. calibration)
-      filterNot(_.equals(OBS_EXP_TIME_KEY)). // don't show observe exp time
-      filterNot(_.equals(INST_EXP_TIME_KEY)). // don't show instrument exp time
-      filterNot(_.equals(TEL_P_KEY)). // don't show offsets
-      filterNot(_.equals(TEL_Q_KEY)). // don't show offsets
-      filterNot(_.getParent().equals(CALIBRATION_KEY)). // calibration settings are not relevant
+      filterNot(k => ExcludedParentKeys.contains(k.getParent)).
+      filterNot(ExcludedKeys.contains).
```

Comment:
A wee tiny thing but if `ExcludedKeys` were a `Set` you could drop the `.contains` since that's that the `Set.apply` does.